File: Android /Android/25:Jan:2017/FishbowlMobileApps/AndroidSDK/FishbowlMobileApps-Jamba-Android-v2.4.1/JMBA_Olo_iOS_SDK/Olo/OloLoyaltyReward.swift
//
//  OloLoyaltyReward.swift
//  Olo
//
//  Created by Eneko Alonso on 6/30/15.
//  Copyright (c) 2015 Hathway, Inc. All rights reserved.
//

import Foundation
import SwiftyJSON

public struct OloLoyaltyReward {

    public var type: String
    public var applied: Bool
    public var label: String
    public var value: Double
    public var reference: String
    public var rewardId: Int64?
    public var membershipId: Int64

    public init(json: JSON) {
        type         = json["type"].stringValue
        applied      = json["applied"].boolValue
        label        = json["label"].stringValue
        value        = json["value"].doubleValue
        reference    = json["reference"].stringValue
        membershipId = json["membershipid"].int64Value
        rewardId     = json["rewardid"].int64
    }
    
}

File: Android /Android/25:Jan:2017/FishbowlMobileApps/AndroidSDK/FishbowlMobileApps-Jamba-Android-v2.4.2/JMBA_Olo_Android_SDK/app/src/main/java/com/wearehathway/apps/olo/Models/OloLoyaltyReward.java
package com.wearehathway.apps.olo.Models;

/**
 * Created by Nauman Afzaal on 03/07/15.
 */
public class OloLoyaltyReward
{
    private String type;
    private boolean applied;
    private String label;
    private double value;
    private String reference;
    private int membershipid;
    private int rewardid;

    public String getType()
    {
        return type;
    }

    public boolean isApplied()
    {
        return applied;
    }

    public String getLabel()
    {
        return label;
    }

    public double getValue()
    {
        return value;
    }

    public String getReference()
    {
        return reference;
    }

    public int getMembershipid()
    {
        return membershipid;
    }

    public int getRewardid()
    {
        return rewardid;
    }
}
